app/views/past_foreign_secretaries/_show_person.html.erb

```diff
@@ -12,28 +12,83 @@
         }
       ]
     } %>
+  </div>
+  <div class="govuk-grid-column-two-thirds">
     <%= render "govuk_publishing_components/components/title", {
-      context: {
-        text: "History",
-      },
-      title: "Past Foreign Secretaries",
+      context: "Past Foreign Secretaries",
+      title: current_person,
+    } %>
+
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: time_active,
+    } %>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: lead_paragraph,
     } %>
   </div>
-</header>
 
-<div class="govuk-grid-row">
-  <div class="historic-people-list govuk-grid-column-one-quarter govuk-!-padding-left-3 govuk-!-padding-right-3">
-    <a href="#contents" class="js-showhide contents-toggle govuk-!-margin-bottom-3">Show past Foreign Secretaries</a>
-    <div id="contents">
-      <ul class="govuk-list past-foreign-secretaries-menu govuk-!-margin-bottom-6">
-        <%= past_foreign_secretary_nav(current_person) %>
-      </ul>
-      <p class="govuk-body-s">
-        <a href="/government/history/past-foreign-secretaries" class="govuk-link">See all past Foreign Secretaries</a>
-      </p>
+  <div class="govuk-grid-column-two-thirds">
+    <div class="person-profile">
+      <div class="person-profile__image">
+        <%= image_tag image, alt: "", loading: "lazy" %>
+      </div>
+      <div class="person-profile__text">
+        <%= image_alt %>
```

Comment:
I'm not sure this is ideal, AT will ignore the image and then hear "Portrait of..." without context.  However if it was on the actual alt it would be "image, Portrait of..." that would make more sense.

I was taking a look at the image card on the gem there's use of `<figure>` and other patterns

Reply:
This choice is based of [our own guidance on image publishing](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/images-and-videos). The specific line:

> Describe your image in the body text, and leave the ‘Alt text’ field blank. Only write a caption if you need one.

A quick test myself does suggest that this might be a little weird for screen reader users as it's as you say, just the caption itself without any context. I'm wondering if it might make more sense to just remove the caption altogether. That way screen readers would just skip over this purely decorative section. Opinions? @nnagewad?

Reply:
> I'm wondering if it might make more sense to just remove the caption altogether. That way screen readers would just skip over this purely decorative section. 

This might work but in a different way - if you change the Content of the caption to remove "Portrait of" this would qualify for [example 3](https://www.w3.org/WAI/tutorials/images/decorative/) as you have an _image with adjacent text alternative_.

Reply:
Updating this content is pretty easy as I'm injecting "portrait of"/"picture of"/"sculpture of" into this. I'm however now a little skeptical as we've then got 2 bits of duplicate content: The h1 and this caption. I don't know if we have other examples where the context of duplicate content makes it ok?